### PR TITLE
Path length 

### PR DESCRIPTION
**Pull request by egolol**
- [x] PathLength, Av

- [x] IN/OUT - PathLength, Av

### CODE REVIEW

File: braph2genesis/measures/_OutPathLengthAv.gen.m [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    .125  0   0   0    0;
+    .2    .5  0   .25  0;
+    .125  10  0   0    0;
+    0     0   0   0    0 
+    ];
+
+known_out_path_length = {Inf};
+
+g = GraphBD('B', A);
+out_path_length = OutPathLengthAv('G', g).get('M');
```
Comment (annacanal) [inline braph2genesis/measures/_OutPathLengthAv.gen.m:54]: out_path_length_av

File: braph2genesis/measures/_PathLength.gen.m [hunk 2]
```diff
@@ -0,0 +1,124 @@
+%% ¡header!
+PathLength < Measure (m, path length) is the graph path length.
+
+%%% ¡description!
+The path length is the average shortest path lengths of one node to all
+other nodes within a layer.
+
```
Comment (annacanal) [inline braph2genesis/measures/_PathLength.gen.m:7]: length*

File: braph2genesis/measures/_OutPathLengthAv.gen.m [hunk 3]
```diff
@@ -0,0 +1,91 @@
+%% ¡header!
+OutPathLengthAv < OutPathLength (m, average out-path length) is the graph average out-path length.
+
+%%% ¡description!
+The average out-path length is the average shortest out-path lengths of one
```
Comment (annacanal) [inline braph2genesis/measures/_OutPathLengthAv.gen.m:5]: the description is the same of OutPathLength, change

File: braph2genesis/measures/_InPathLengthAv.gen.m [hunk 4]
```diff
@@ -0,0 +1,91 @@
+%% ¡header!
+InPathLengthAv < InPathLength (m, average in-path length) is the graph average in-path length.
+
+%%% ¡description!
+The average in-path length is the average shortest in-path lengths of one
```
Comment (annacanal) [inline braph2genesis/measures/_InPathLengthAv.gen.m:5]: the description is the same of InPathLength, change

**Review by annacanal — CHANGES_REQUESTED** [review]
I would add as well test for GraphWU and GraphBU in Path length, and so. since we had them on the version in folder braph2

**Review by annacanal — APPROVED** [review]

**Review by c-yuwei — APPROVED** [review]

File: braph2genesis/measures/_InPathLength.gen.m [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+%%% ¡compatible_graphs!
+GraphBD
+GraphWD
+MultiplexGraphBD
+MultiplexGraphWD
+
+%% ¡props!
+%%% ¡prop! 
+rule (metadata, OPTION)
```
Comment (giovannivolpe) [inline braph2genesis/measures/_InPathLength.gen.m:25]: complete definition of prop

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
please, complete definitions of props

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**